releases/release-1.27/release-notes/maps/pr-114494-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 114494
+releasenote:
+  text: The ReadWriteOncePod feature gate has been graduated to beta.
```

Comment:
```suggestion
  text: The `ReadWriteOncePod` feature gate has been graduated to beta.
```

Reply:
Yeah, something very strange happened during my rebase of accepting her changes last night. I guess I screwed up and quite a few of them were disgarded. I'm fixing now.